dubbo-config/dubbo-config-spring/pom.xml

```diff
@@ -152,6 +152,11 @@
                     <artifactId>junit-vintage-engine</artifactId>
                     <groupId>org.junit.vintage</groupId>
                 </exclusion>
+                <!-- Fix the bug of log4j refer:https://github.com/apache/logging-log4j2/pull/608 -->
+                <exclusion>
+                    <groupId>org.apache.logging.log4j</groupId>
```

Comment:
`log4j-core` hava security vulnerability only need to upgrade `log4j-api` ?

Reply:
For compatibility consideration, I think upgrading `l0g4j-api` is fine.